framework/src/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -1817,6 +1851,12 @@ public AnnotationMirror createArrayLenRangeAnnotation(Range range) {
         }
     }
 
+    public AnnotationMirror createErrorBottom(String errorKey) {
+        AnnotationBuilder builder = new AnnotationBuilder(processingEnv, BottomVal.class);
```

Comment:
Currently, "from.greater.than.to" is the only error key. So you could create that annotation once and then use it directly at the call sites to this method.